Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3303,8 +3310,13 @@ - (void)uiDatabaseDidUpdate:(NSNotification *)notification
             case YapDatabaseViewChangeUpdate: {
                 YapCollectionKey *collectionKey = rowChange.collectionKey;
                 if (collectionKey.key) {
-                    ConversationViewItem *viewItem = self.viewItemCache[collectionKey.key];
-                    [self reloadInteractionForViewItem:viewItem];
+                    ConversationViewItem *_Nullable viewItem = self.viewItemCache[collectionKey.key];
+                    if (viewItem) {
+                        [self reloadInteractionForViewItem:viewItem];
+                    }
+                } else if (rowChange.indexPath && rowChange.originalIndex < self.viewItems.count) {
+                    // Do nothing, this is a pseudo-update generated due to
```

Comment:
nit: can you add an assert:

OWSAssert(rowChange.changes == YapDatabaseViewChangedDependency)

Reply:
Good idea, done.